### ISSUE TITLE
Demo

### ISSUE THREAD

**Issue by runyontr** [defenseunicorns/sbom-cli#1]
For a particular version of Big Bang, I would like to be able to:

1) Identify all the images and use syft to create an SBOM for each.
- [ ] Gatekeeper
- [ ] Istio Operator
- [ ] Istio Controlplane
- [ ] Monitoring
- [ ] ECK Operator
- [ ] Elasticsearch Kibana
- [ ] Cluster Auditor
- [ ] Twistlock
- [ ] Fluentbit

2) For each chart in Core, update the chart.yaml to work with our syntax and generate an SBOM from that.  Future work will add depth based on chart dependency
3) For Big Bang, take each chart's sbom and put it into BB's BOM with as a dependency.

**Comment by runyontr** [comment]
Viewing more than 3 levels is currently blocked by the front end UI issue:

https://github.com/DependencyTrack/frontend/issues/85
